lib/papelillo/mocks/http_mock.ex

```diff
@@ -0,0 +1,23 @@
+defmodule Papelillo.Mocks.HttpMock do
+  alias Papelillo.Stub.Providers.HttpClient
```

Comment:
I don't see the need for this separate module, you can put directly the body of the response here instead of having to call another module that does not have any logic in there and is also not reusable.